recipes/tensorflow-quantum/meta.yaml

```diff
@@ -0,0 +1,76 @@
+{% set name = "tensorflow-quantum" %}
+{% set version = "0.5.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/tensorflow/quantum/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: c6a4a72366334673c38cd06329f17889278db4dba8060beb891ef3c489fb72a8
+
+build:
+  number: 0
+  skip: true   # [ py<36 or win]
+  script:
+    - export TF_NEED_CUDA=0
+    - export TF_CUDA_VERSION=10.1
+    - ./configure.sh
+    - bazel build -c opt --cxxopt="-O3" --cxxopt="-march=native" --cxxopt="-D_GLIBCXX_USE_CXX11_ABI=0" release:build_pip_package
+    - bazel-bin/release/build_pip_package /tmp/tfquantum/
+    - ${PYTHON} -m pip install {{ name|lower }}-{{ version }}-py3-none-any.whl -vv --no-deps --no-index
+    - ./scripts/test_all.sh
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - bazel
+  host:
+    - future >=0.17.1
+    - mock
+    - numpy 1.19.*
+    - pip
+    - python
+    - setuptools
+    - six
+    - tensorflow >=2.5.1
+    - wheel
+  run:
+    - python
+    - cirq-core >=0.13.1
+    - cirq-google >=0.13.1
+    - sympy >=1.8
+    - {{ pin_compatible('numpy') }}
+    - nbconvert >=5.6.1
+    - nbformat >=4.4.0
+    - pylint >=2.4.4
+    - yapf >=0.28.0
+    - tensorflow >=2.5.1
+    # Needed for compatibility with cirq program protos.
```

Comment:
Should this then be in the cirq recipes?

Reply:
The comment is from upstream. Since both libraries are from Google I think they would've put it there if needed but I'll ask upstream.

Reply:
From my understanding, `googleapis-common-protos` is used to enable common `protocol buffers` amongst multiple Google packages. Therefore, this is required here as it's interacting with `cirq`.